### DISCUSSION TITLE
Trigger not fires when db contains trigger data

### DISCUSSION THREAD

**Discussion by Braidon** [quartznet/quartznet#595] [General] [answered]
### Expected behavior
Trigget must fire after program restart if trigger start time was lost.

### Actual behavior
When I re-create quartz tables at start everything works fine.
Second run without "cleaning" db not causes trigger to fire.

### Steps to reproduce
1. run app (it will work)
2. rem  CreateQuartzNetTables line and run again. it won't work
Maybe I missed something?

Test project:
[QuartzNetAdoJobStoreSQLite.zip](https://github.com/quartznet/quartznet/files/1743905/QuartzNetAdoJobStoreSQLite.zip)
Thank you

**Comment by lahma** [comment, marked as answer]
I unfortunately don't have the time to comb code samples looking for all possible problems. I quickly checked your sample and here are a few pointers:

* You don't await or use (GetAwait.GetResult()) for all Task-returning methods
* Json.NET version is 11 when Quartz uses 10, there's no proper binding redirect
* Common.Logging is not needed with Quartz.NET 3
* Make sure you log errors and at least print them to console
* Try running under debugger with catch all exceptions enabled to see where problems might rise
